samples/bluetooth/scanning_while_connecting/boards/nrf54h20dk_nrf54h20_cpuapp.conf

```diff
@@ -10,3 +10,6 @@ CONFIG_PSA_SSF_CRYPTO_CLIENT=y
 CONFIG_SSF_PSA_CRYPTO_SERVICE_ENABLED=y
 CONFIG_MBEDTLS_PSA_CRYPTO_C=y
 CONFIG_NRF_SECURITY=y
+
+# Configure app core firmware to enable radio core
+CONFIG_SOC_NRF54H20_CPURAD_ENABLE=y
```

Comment:
This looks like it fixing the symptom but not the root cause.
I'm wondering why the radio code is not enabled by default when enabling BT.